### DISCUSSION TITLE
카카오 refresh가 안돼요

### DISCUSSION THREAD

**Discussion by 1q2w3e4r-1** [changdoc/naver-vaccine-macro#164] [Q&A] [answered]
![112121](https://user-images.githubusercontent.com/88218812/127725863-9e81be6f-df69-40a0-9706-a051be26bb08.PNG)
카카오 업데이트가 눌려지기는 하는데 화살표가 끝까지 안돌아가면서 시간이 업데이트가 안됩니다 F12누르고 들어가보면 사진처럼 error occurred가 뜹니다 자동업데이트 눌러놓고 업데이트 누르면 업데이트가 되기는 하는데 매크로로 업데이트가 되야 매크로가 작동하는지 잔여백신이 떠도 반응이 없는것 같습니다
1. 네이버에 원하는곳 지도 옮겨놓고 F12누르고 새로고침해서 graphql 뜬거 붙여넣고 병원생성저장
2. 카카오지도 들어가서 네이버와 비슷하게 옮겨놓고 잔여백신 있음 체크 해제한 상태에서 F12누르고 새로고침해서 left_count-by-coords 뜬거 붙여넣고 저장
3. 잔여백신 있음 체크 해제한 상태에서 자동새로고침 시작
이렇게 진행했습니다
아직 테스트중인 기능이라 작동을 안하는 기능일까요 ㅠㅠ 가능하면 카카오로 확인하고 네이버로 예약하고 싶은데 안될까요?
네이버웨일에서 개발자모드로 추가해서 사용했습니다

**Comment by changdoc** [comment, marked as answer]
카카오맵쪽은 네이버 맵쪽이랑 동작방식이 조금 다른데요.
2번 항목에 넣어야하는 데이터가 다른걸 넣으셔서 그런거 같아요.
![image](https://user-images.githubusercontent.com/1879494/127726331-5cc3e6ca-052b-4b05-837a-7d4e89421cc6.png)
이런 형식인데.. 추출 방법은
바로 가이드 작성을 하고 다시 댓글 남기겠습니다.

**Comment by changdoc** [comment]
카카오 맵 쪽 동작 가이드 추가했습니다.
https://github.com/changdoc/naver-vaccine-macro/issues/166

**Reply by 1q2w3e4r-1** [reply]
감사합니다
가이드 따라서 데이터 넣었더니 오류안뜨네요
